app/domain/authentication/authn_gcp/authenticator.rb

```diff
@@ -5,27 +5,37 @@ module AuthnGcp
 
     Authenticator = CommandClass.new(
```

Comment:
this is out of scope from your PR but it would be good to add here a comment that part of the authentication process for authn-gcp is to validate the token but it happens elsewhere. just for clarity. something similar to the comment we have for oidc: https://github.com/cyberark/conjur/blob/master/app/domain/authentication/authn_oidc/authenticator.rb#L11

Reply:
done